robocup_ws/src/Planner/src/player_controller.py

```diff
@@ -19,46 +19,52 @@
 #Having us just worry about adding extra functionalities instead of coding all over again
 
 class PlayerController:  #(Robot)
+    goal_threshold = 0.2
```

Comment:
if you lower this value (0.17 or 0.15) would you get the same result as changing it in ```ball_model```?